### PR TITLE
fix: various UI tweaks and cleanup

### PR DESCRIPTION
**Pull request by bryce-seifert**
Some fixes after coreui update, and some other general ui tweaks

### CODE REVIEW

File: webui/src/Triggers/index.tsx [hunk 1]
```diff
@@ -329,10 +329,10 @@ function TriggersTableRow({ controlId, item, editItem, moveTrigger, isSelected }
 			</td>
 			<td className="action-buttons">
 				<CButtonGroup>
-					<CButton color="white" onClick={doClone} title="Clone">
+					<CButton color="white" onClick={doClone} title="Clone" style={{ padding: 4 }}>
```
Comment (Julusian) [inline webui/src/Triggers/index.tsx]: I've pushed a change that should negate the need for these https://github.com/bitfocus/companion/commit/8fddb40ba120c1f228ec6fa58ca2e5f253c5f4e0

**Comment by Julusian** [comment]
personally, this area feels a bit bare now (for a stable release). maybe doesn't need to be changed as part of this though
image

File: webui/src/Triggers/index.tsx (hunk 1 above)
Comment (bryce-seifert) [reply webui/src/Triggers/index.tsx]: Got it, I just reverted that.

**Comment by bryce-seifert** [comment]
Good point, I agree. I mostly tested it with in tandem with an Installation Name, so it looks more full there. But I bet currently the amount of people using a custom name is low.

**Comment by Julusian** [comment]
I think having the build visible is really useful for beta builds, just saying '3.4.0' will make it hard to tell what is a stable release and what is a beta. So I've put that back for any non-stable build

**Comment by Julusian** [comment]
> Good point, I agree. I mostly tested it with in tandem with an Installation Name, so it looks more full there. But I bet currently the amount of people using a custom name is low.

I've been wondering for a while about simply pushing it to the right corner instead. But not enough to try it

**Comment by Julusian** [comment]
github is complaining about a merge conflict, other than that I am happy to merge this

**Comment by bryce-seifert** [comment]
Should be good now, looks like we both had the same idea about the zoom panel closing

**Pull request merged**